app/build.gradle

```
@@ -34,10 +35,14 @@ android {
     buildFeatures {
         viewBinding true
     }
+
+    dataBinding {
+        enabled = true
+    }
```

안드로이드 스튜디오 4.0 버전 이상부터는 Gradle에서
```suggestion
buildFeatures {
   dataBinding true
}
```
이런식으로 바인딩을 선언합니다~!!